tests/hypothesis_strategies/parameters.py

```diff
@@ -75,14 +76,20 @@ def custom_descriptors(draw: st.DrawFn):
 @st.composite
 def numerical_discrete_parameter(
     draw: st.DrawFn,
+    min_value: Optional[float] = None,
+    max_value: Optional[float] = None,
 ):
     """Generate :class:`baybe.parameters.numerical.NumericalDiscreteParameter`."""
     name = draw(parameter_name)
     values = draw(
         st.lists(
             st.one_of(
```

Comment:
is the `one_of` still required since you kicked ints?

Reply:
Nope, excellent catch, thanks! Fixed